demo/.babelrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "sourceType": "unambiguous",
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "chrome": 100
+        }
+      }
+    ],
+    "@babel/preset-react"
+  ],
+  "plugins": []
+}
```

Comment:
Sorry, can you explain why this babel config is required?

Reply:
Storybook 6 provided some babel config. v7 no longer does, and you have to provide your own to render the React components and whatnot.